tests/lightning_qubit/test_adjoint_jacobian_class.py

```diff
@@ -23,11 +23,23 @@
 from pennylane.tape import QuantumScript
 from scipy.stats import unitary_group
 
-from pennylane_lightning.lightning_qubit._adjoint_jacobian import LightningAdjointJacobian
-from pennylane_lightning.lightning_qubit._state_vector import LightningStateVector
+if device_name == "lightning.qubit":
+    from pennylane_lightning.lightning_qubit._adjoint_jacobian import LightningAdjointJacobian
+    from pennylane_lightning.lightning_qubit._state_vector import LightningStateVector
 
-if device_name != "lightning.qubit":
-    pytest.skip("Exclusive tests for lightning.qubit. Skipping.", allow_module_level=True)
+if device_name == "lightning.kokkos":
+    from pennylane_lightning.lightning_kokkos._adjoint_jacobian import (
+        LightningKokkosAdjointJacobian as LightningAdjointJacobian,
+    )
+    from pennylane_lightning.lightning_kokkos._state_vector import (
+        LightningKokkosStateVector as LightningStateVector,
+    )
```

Comment:
These lines will move to `conftest.py` when we have done the migrations of all devices to the new device API

Reply:
Sounds good!